packages/components/src/List/ListComposition/ColumnChooser/ColumnChooserHeaderButton.component.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ButtonIconFloating } from '@talend/design-system';
+
+import ColumnChooser from './ColumnChooser.component';
+import ColumnChooserButton from '../../Toolbar/ColumnChooserButton';
+import theme from './ColumnChooserHeaderButton.scss';
+
+export default function ColumnChooserHeaderButton(props) {
+	return (
+		<div className={theme.columnChooserHeaderButtonContainer}>
+			<ColumnChooser {...props} buttonRenderer={ButtonIconFloating} />
+		</div>
+	);
+}
+
+ColumnChooserHeaderButton.propTypes = PropTypes.oneOfType(
```

Comment:
not sure of these proptypes 🤔

Reply:
What do you mean? Not a good practice to have property with 2 possible types? Or you're not sure about current types?